service/debugger/debugger.go

```diff
@@ -182,16 +193,45 @@ func (d *Debugger) CreateBreakpoint(requestedBp *api.Breakpoint) (*api.Breakpoin
 	if err != nil {
 		return nil, err
 	}
-	bp.Tracepoint = requestedBp.Tracepoint
-	bp.Goroutine = requestedBp.Goroutine
-	bp.Stacktrace = requestedBp.Stacktrace
-	bp.Variables = requestedBp.Variables
-	bp.Cond = nil
+	if err := copyBreakpointInfo(bp, requestedBp); err != nil {
+		d.process.ClearBreakpoint(bp.Addr)
```

Comment:
Clearing the breakpoint could error out, we should surface that error but provide a meaningful message so the user isn't confused as to why a `ClearBreakpoint` error happened when they were trying to set a breakpoint. Either that, or try and parse the conditional expression earlier and surface an invalid condition error earlier, before the breakpoint is ever set.